### PR TITLE
Footer added to the bottom of page

### PR DESCRIPTION
**Pull request by Sigouin**
#### Summary

Added a Footer that shows current Version number of the application, as well as links to less important pages.

#### Context
User wanted to see a footer at the bottom of the page so that they could easily get an overview and navigate to less important pages.

#### Testing
Added tests to Cypress to make sure all links and Version number work correctly on both desktop and mobile resolutions.

#### Checklist
- [ x ] All tests pass successfully.
- [ x ] Code follows the project's style guide and conventions.
- [ x ] This pull request is ready for review.

### CODE REVIEW

**Review by mentor-tara — APPROVED** [review]
Go go go :smile:[](prolog-8)

File: features/layout/footer-navigation/footer-navigation.module.scss [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  align-items: center;
+  justify-content: center;
+  align-self: stretch;
+  height: 60px;
+
+  @media (max-width: breakpoint.$mobile) {
+    height: 166px;
+  }
+
+  .container {
```
Comment (jkettmann) [inline features/layout/footer-navigation/footer-navigation.module.scss:20]: I wouldn't nest the classes this deeply. It get's hard to understand if and how the parent classes affect the child classes in some way. Here the `.container` could probably be at the root level. Same as the nested other classes

File: features/layout/footer-navigation/footer-navigation.module.scss [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+          color: color.$gray-500;
+          text-decoration: none;
+        }
+      }
+
+      .logoContainer {
+        display: flex;
+
+        @media (min-width: breakpoint.$desktop) {
+          width: 33%;
```
Comment (jkettmann) [inline features/layout/footer-navigation/footer-navigation.module.scss:56]: Another approach that is a bit safer is to use `flex: 1` and remove the width on the logo container. You probably won't even need the media query then. At least not for the width.

File: features/layout/footer-navigation/footer-navigation.module.scss [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    align-self: stretch;
+    padding: 0 space.$s8;
+    gap: space.$s8;
+
+    .footerContent {
+      display: flex;
+      justify-content: space-between;
```
Comment (jkettmann) [inline features/layout/footer-navigation/footer-navigation.module.scss:30]: You probably realized that space-between doesn't distribute the space equally among the elements. Instead you can remove `justify-content` here and go for the `flex: 1` approach mentioned in the other comment

File: features/layout/footer-navigation/footer-navigation.module.scss [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+@use "@styles/misc";
+
+.footer {
+  background-color: color.$gray-50;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  align-self: stretch;
+  height: 60px;
```
Comment (jkettmann) [inline features/layout/footer-navigation/footer-navigation.module.scss:14]: Slight consistency issue: the rest of the codebase uses rem unit

**Pull request merged**